### PR TITLE
Constant variables to upper case & snake case

### PR DESCRIPTION
**Pull request by AnaLucyGDL**
Change the constant variables to the requested format

### CODE REVIEW

**Review by tris460 — CHANGES_REQUESTED** [review]

File: scrapperFunctions/index.js [hunk 1]
```diff
@@ -2,22 +2,22 @@
 'use strict';
 /**
  * Initialize the instances
- * @param {puppeteer} browser Instance of puppeteer
+ * @param {puppeteer} BROWSER Instance of puppeteer
  */
-function ScrapPage(browser) {
-  this.browser = browser;
-  this.page = browser.page;
+function SCRAP_PAGE(BROWSER) {
```
[the remark this answers is not in the document]
Comment (wolftrax5) [reply scrapperFunctions/index.js:7]: The function not need to be in upercase, is not a constant, the name of this one not need change

File: scrapperFunctions/index.js [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
+  this.BROWSER = BROWSER;
+  this.page = BROWSER.page;
```
[the remark this answers is not in the document]
Comment (wolftrax5) [reply scrapperFunctions/index.js:9]: Same here BROWSER should be on lowercase

File: webScraping/cfeScrapper.js [hunk 3]
```diff
@@ -9,16 +9,16 @@
...
-const URLPage = 'https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/';
-const idInput = '#descProc';
-const idButton = '#buscar';
-const waitingTime = 2000;
+const PUPPETEER = require('puppeteer');
+const SCRAP_PAGE = require('../scrapperFunctions/index');
+const URL_PAGE = 'https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/';
+const ID_INPUT = '#descProc';
+const ID_BUTTON = '#buscar';
+const WAITING_TIME = 2000;
```
Comment (wolftrax5) [inline webScraping/cfeScrapper.js:17]: THIS IS THE CORRECT USE OF UPERCASE TO CONSTS

File: webScraping/cfeScrapper.js [hunk 4]
```diff
@@ -9,16 +9,16 @@
...
 // const ROUTE = './assets/Data-From-Table.json';
-const nextPageBtn = 'div.row a.k-link span.k-i-arrow-e';
-const tableSelector = 'table.k-selectable';
-const rowSelector = '#totProc';
+const NEXT_PAGE_BTN = 'div.row a.k-link span.k-i-arrow-e';
+const TABLE_SELECTOR = 'table.k-selectable';
+const ROW_SELECTOR = '#totProc';
```
Comment (wolftrax5) [inline webScraping/cfeScrapper.js:21]: Here too this is the correct use of const with upercase

File: webScraping/cfeScrapper.js [hunk 5]
```diff
@@ -35,11 +35,11 @@ function SCRAPPER(text) {
  */
 // eslint-disable-next-line space-before-function-paren
 SCRAPPER.prototype.newBrowser = async function () {
-  const browser = await puppeteer.launch({
+  const BROWSER = await PUPPETEER.launch({
     args: ['--no-sandbox', '--disable-setuid-sandbox'],
   });
   console.log('Opening a new browser...');
-  return browser;
+  return BROWSER;
```
Comment (wolftrax5) [inline webScraping/cfeScrapper.js:42]: Bad use of uperCase

**Review by wolftrax5 — CHANGES_REQUESTED** [review]
No because we use the keyword "const" means need to be in UPERCASE, pls change it only where is need it.

File: utils/validator.js [hunk 6]
```diff
@@ -7,20 +7,20 @@ const { isFileHasLessHoursUpdate } = require('./files');
...
+function validation() { }
 
 /**
  * Compares the date´s file and today to check if doScraping or not
  */
 // eslint-disable-next-line space-before-function-paren
-Validation.prototype.isFileLastUpdateIn = function (path, hours = 24) {
-  const validateFile = isFileHasLessHoursUpdate(hours);
+validation.prototype.isFileLastUpdateIn = function (path, hours = 24) {
+  const validationfile = isFileHasLessHoursUpdate(hours);
```
Comment (wolftrax5) [inline utils/validator.js:19]: for validationfile is correct use the camel case validationFile, but Is ok

**Review by wolftrax5 — APPROVED** [review]

**Review by tris460 — APPROVED** [review]

**Pull request merged**